### PR TITLE
Generator for honeybadger

### PR DESCRIPTION
**Pull request by mausamp**

### CODE REVIEW

File: lib/generators/boring/honeybadger/install/install_generator.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+      def honeybadger_api_key
+        if options[:use_env_variable]
+          "ENV['HONEYBADGER_API_KEY']"
+        else
+          "Rails.application.credentials.dig(:honeybadger, :api_key)"
+        end
+      end
+
+      def display_alert_message
```
Comment (abhaynikam) [inline lib/generators/boring/honeybadger/install/install_generator.rb:40]: can you move this to readme.md file within the generator? you can refer other generator to display post install messages.

File: lib/generators/boring/honeybadger/install/install_generator.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        @api_key = honeybadger_api_key
+
+        template 'honeybadger.yml', 'config/honeybadger.yml'
+
+        display_alert_message
+      end
+
+      private
+
+      def honeybadger_api_key
```
Comment (abhaynikam) [inline lib/generators/boring/honeybadger/install/install_generator.rb:32]: can you fix the indentation here?

**Review by abhaynikam — APPROVED** [review]
Left some comments, but otherwise looks good to me.

**Comment by abhaynikam** [comment]
@mausamp can you rebase your branch?

File: lib/generators/boring/honeybadger/install/install_generator.rb (hunk 2 above)
Comment (mausamp) [reply lib/generators/boring/honeybadger/install/install_generator.rb:38]: Can you please specify more about the indentation, as per my understanding it looks good.

**Pull request merged**